src/main/java/frc/robot/RobotContainer.java

```diff
@@ -187,23 +185,28 @@ private void configureBindings() {
                 }));
 
     controller
-        .x()
+        .b()
```

Comment:
My 2c, I think you're going to be happier if every position is in armPositions (default + source + reef). So it'd look something like this:

```
.b().onTrue(setArmPositions(SOURCE_ARM_POSITION));
.x().onTrue(setArmPositions(this.armPosition));
default(setArmPositions(DEFAULT_ARM_POSITION));


public setArmPositions(ArmPositionEnum position) {
  wrist.setGoalPosition(armPositions[position].wrist);
  pivot.setPosition(armPositions[position].pivot);
  elevator.setPosition(armPositions[position].elevator);
}
```